spring-boot-admin-server-ui/src/main/frontend/i18n/default.de.json

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
Would it be somehow possible to move the translations not in a central file, but ins separate files per view and closer to the views and stitched to gether via `require.conext`? (eg. https://github.com/codecentric/spring-boot-admin/blob/master/spring-boot-admin-server-ui/src/main/frontend/components/index.js#L21)

Reply:
I will try to do that.

Reply:
Would it be okay for you to have a file called `i18n.json` aside each and every view? Or would you prefer to have it split up by language as well like `i18n.en.json`?

Reply:
I've no objections on this. I leave this up to you.